### ISSUE TITLE
Backend assignment creation bug

### ISSUE THREAD

**Issue by theknickerbocker** [Theknickerbocker/UB_Grader#140]
If the assignment name has spaces in it then 2 directories are made (1 w/ spaces 1 w/ %20). One directory holds Tango stuff, the other holds submissions dir. Then submissions never are put in that submissions dir

**Comment by theknickerbocker** [comment]
![screen shot 2017-10-20 at 12 28 05 pm](https://user-images.githubusercontent.com/9044082/31831639-28c5ccb4-b592-11e7-8486-926068a99e73.png)

**Comment by adhishchugh** [comment]
@Theknickerbocker Yeah we need to add some error handling here. You shouldn't be able to create an assignment if assignment_num has spaces. It's essentially a slug. Only name can have spaces.

**Comment by theknickerbocker** [comment]
Ok, also directory naming convention needs to be changed because rn if I create an assignment with the same name in 2 different categories (like Q2 in HW1 and Q2 in HW2) then the 2nd directory isn't created. This also means that the paths of the two assignments overlap

**Comment by theknickerbocker** [comment]
Honestly I don't think making users specify assignment nums makes much sense

**Comment by theknickerbocker** [comment]
@adhishchugh This was fixed right?

**Comment by adhishchugh** [comment]
@Theknickerbocker Ah, no. Checking for duplicates is fixed but we don't slugify anything.

**Comment by theknickerbocker** [comment]
Ok well looks like we enforce it on our side so that should cover us for now

**Comment by adhishchugh** [comment]
Okay great!
